README.md

```diff
@@ -245,22 +246,45 @@ After which we'd be able to connect to the App server at HTTPS port 19998.
 
 When the zLUX App Server has started, one of the last messages you will see as bootstrapping completes is that the server is listening on the HTTP/s port. At this time, you should be able to use the server.
 
-### 7. Connect in a browser
+### 7. Connect to the Zowe Desktop via a browser
 Now that ZSS & the zLUX App Server are both started, you can access this instance by pointing your web browser to the zLUX App Server.
-In this example, the address you will want to go to first is the location of the window management App - Zowe Desktop.
+In this example, the address you will want to go to first is the location of the window management App - the Zowe Desktop.
```

Comment:
May want to remove this, is pseudo-confusing to new users. No one thinks of a desktop as an "app" in the modern world, thoughts?